assets/scss/components/_ie-fixes.scss

```diff
@@ -0,0 +1,95 @@
+html.ie9 {
+    .Header {
+        height: 50px;
+
+        > * {
+            float: left;
+        }
+    }
+
+    .Frame-handle {
+        display: none;
+    }
+
+    .Frame-body {
+        height: auto;
+        width: 100%;
+    }
+    .Frame-inner {
+        margin-top: 50px;
+    }
+    .Frame-panel {
+        overflow: auto;
+    }
+    .Frame-panel--sidebar {
```

Comment:
@tlenex is there a particular reason why you chose to implement this as a toggle-able, absolutely positioned 'menu' panel instead of a fixed-width sidebar for IE9? I have to say my preference would be with a simple fixed-width sidebar (even if it we lose the show/hide ability). Obviously there is less horizontal space for viewing the components in that case but the overlaid menu doesn't feel right to me (and, as I wasn't expecting it, actually thought it was a bug initially). But I would be interested to hear your thoughts.